src/numbers/Properties.java

```diff
@@ -0,0 +1,12 @@
+package numbers;
+
+public class Properties {
+
+    public static final String[] AVAILABLE_PROPERTIES
```

Comment:
Why array?

Reply:
Because there is no need to use features of more complicated collections.